addon.js

```diff
@@ -8,7 +8,14 @@
     '#statusbar.light { filter:none !important; }',
     '#windows .appWindow .titlebar { background: #333333 !important }',
     '#windows .appWindow.homescreen { display:none !important; animation: none !important }',
-    '#software-home-ring { outline: none; background: no-repeat center/70% url(' + browser.extension.getURL('assets/Home.png') + '); }',
+    '#software-home-ring {',
+    'outline: none;',
+    '--ring-height-width: inherit;',
```

Comment:
Something native on the existing button, I'll figure out what it is used for and use the property instead.